foundations/html_css/css-foundations.md

```diff
@@ -13,7 +13,7 @@ This section contains a general overview of topics that you will learn in this l
 
 ### Basic Syntax
 
-At the most basic level, CSS is made up of various rules. These rules are made up of a selector (more on this in a bit) and a semi-colon separated list of declarations, with each of those declarations being made up of a property:value pair.
+At the most basic level, CSS is made up of various rules. These rules are made up of a selector (more on this in a bit) and a semi-colon separated list of declarations, with each of those declarations being made up of a property:value pair. To understand more about the various values and their types possible, click the values and units link in the additional resources section towards the end of this page.
```

Comment:
We may not need this verbiage here. Typically we don't mention an additional resource exists in the middle of a lesson. I think one issue with doing so could be that it might cause users to shift to that resource and go down a rabbit hole of info they don't necessarily *need* to understand at that moment.